app/views/reports/bch/registries/view-lab-registry.js

```diff
@@ -51,6 +51,13 @@ export default ['$scope', 'searchService', '$element', '$rootScope', 'translatio
 				}, 200)
 			});
 		}
+		$scope.removeLabFromStart= function (str) {
+			if (str.startsWith('LAB -')) {
+				return str.slice(6);
+			} else {
+				return str;
+			}
```

Comment:
@blaisefonseca will say that i'm always saying that but...
you can also use regular expresssion `/^LAB -/`. if the pattern match it will be replaced by empty string!
you can also have variant with variable number of spaces or invariant case eg:
``` javascript
/^LAB\s*-\s*/i
```
eg: https://regexr.com/7u9si

```suggestion
			return str.replace(/^LAB\s*-\s*/i, '');
```